src/CMakeLists.txt

```diff
@@ -540,7 +559,7 @@ find_package(Pugixml 1.2 REQUIRED)
 include_directories(SYSTEM ${Pugixml_INCLUDE_DIRS})
 list(APPEND LIBS ${Pugixml_LIBRARIES})
 
-if(NOT SOURCE_PACKAGE AND NOT APPLE)
+if(NOT SOURCE_PACKAGE AND NOT APPLE AND NOT WIN32)
```

Comment:
If there is no SOURCE_PACKAGE, then there is no -Werror at all.
Maybe i'm missing something, which -Werror does turn them into errors for you? The one that you pass manually?